runtime/src/accounts_db.rs

```diff
@@ -493,11 +493,15 @@ impl AccountStorageEntry {
         }
     }
 
-    fn remove_account(&self, num_bytes: usize) -> usize {
+    pub fn accounts(&self, offset: usize) -> Vec<StoredAccountMeta> {
+        self.accounts.accounts(offset)
+    }
+
+    fn remove_account(&self, num_bytes: usize, reset_accounts: bool) -> usize {
```

Comment:
nits: well, I always bothered about naming of `reset` in general. Somewhat confusingly, `accounts` here doesn't mean some kind of collections of `Accounts`, but rather specifically referring to the `.accounts` field, which in turn I think should be named just like `.append_vec`.

how about these renames along line of `rewind_append_vec`, `truncate_store`, `reset_storage`, `clear_accounts`?

Reply:
For example, maybe future me must wonder what `reset_accounts` mean while reading this code residing at the way-higher level in call-graph wise:

https://github.com/solana-labs/solana/pull/15022/files#r569300084

Reply:
Yea. I think I'll do this in a follow-up.